src/imageio.cs

```diff
@@ -901,6 +901,734 @@ interface CGImageProperties {
 		[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
 		[Field ("kCGImagePropertyFileContentsDictionary")]
 		NSString FileContentsDictionary { get; }
+		
+		[Mac (10, 9)][iOS (11,0)]
+		[Field ("kCGImagePropertyOpenEXRDictionary")]
+		NSString OpenExrDictionary { get; }
+
+		[Mac (10, 13, 4)][iOS (11, 3)]
```

Comment:
do we need `onlyOn64: true` here? and on the ones below?

Reply:
it's an old framework so there's a good chance the constants are in 32 bits only
intro should spot such cases